bin/run

```diff
@@ -128,6 +128,10 @@ init_variables() {
     airflow variables -s "surveygizmo_api_secret" "tapsekret"
     airflow variables -s "anomdtct_spreadsheet_id" "secret-id"
     airflow variables -s "anomdtct_spreadsheet_api_key" "secret-key"
+
+    airflow variables -s "taar_bigtable_instance_id" "taar_bigtable_instance_id"
+    airflow variables -s "taar_etl_storage_bucket" "taar_etl_storage_bucket"
+    airflow variables -s "taar_gcp_project_id" "taar_gcp_project_id"
 }
```

Comment:
add taar_dataflow_subnetwork now that you've added it elsewhere

Reply:
this will probably fix your cirlceci test environment check as well

Reply:
Fixed in ec9155412944